src/applications/claims-status/containers/YourClaimsPageV2.jsx

```diff
@@ -284,7 +291,7 @@ class YourClaimsPageV2 extends React.Component {
             <h1 className="claims-container-title">
               Check your claim, decision review, or appeal status
             </h1>
-            <va-on-this-page />
+            <va-on-this-page uswds="false" />
```

Comment:
`va-on-this-page` doesn't have a `uswds` property.

Reply:
yes so even though it doesnt have it yet, we still wanted to add it and set it to false since all components are supposed to be updated to V3 by feb 16.